contracts/modules/staking/StakingPool.sol

```diff
@@ -886,6 +891,22 @@ contract StakingPool is IStakingPool, ERC721 {
     return (allocatedCapacities, allocatedCapacity);
   }
 
+  function getCurrentTotalCapacities(uint productId) public view returns (uint[] memory totalCapacities, uint totalCapacity) {
+    uint firstTrancheIdToUse = block.timestamp / TRANCHE_DURATION;
+
+    uint[] memory productIds = new uint[](1);
+    productIds[0] = productId;
+    (uint24 globalCapacityRatio, uint16[] memory capacityReductionRatio) = ICover(coverContract).getCapacityRatios(productIds);
```

Comment:
The `getCapacityRatios` returned parameters are casted to uints below. Let's change the interface to return `uint`s
directly (will probably save some gas internally)